src/browser-lib/parcel.component.ts

```diff
@@ -0,0 +1,142 @@
+import { Component, OnInit, OnDestroy, Input, OnChanges, ViewChild, ElementRef } from '@angular/core';
+
+
+@Component({
+  selector: 'parcel',
+  template: `
+  <div #parcelDiv>Parcel works!</div>
+  `
+})
+export class ParcelComponent implements OnInit, OnDestroy, OnChanges {
+  @ViewChild('parcelDiv') parcelDiv: ElementRef;
+  @Input() config: any; // import { ParcelConfig } from 'single-spa';
+  @Input() mountParcel: any;
+  @Input() onParcelMount: () => void;
+  @Input() wrapWith = 'div';
+  @Input() customProps: any;
+  @Input() appendTo: any;
+  @Input() handleError = err => console.error(err);
+
+
+  createdDomElement: any;
+
+  hasError: boolean;
+
+  unmounted: any;
+  nextThingToDo: Promise<any>;
+
+  title = 'utente';
+
+  parcel: any;
+
+  constructor() {
+  }
+
+  ngOnInit() {
+    if (!this.config) {
+      throw new Error(`single-spa-react's Parcel component requires the 'config' prop to either be a parcel config or a loading function that returns a promise. See https://github.com/CanopyTax/single-spa-react`)
```

Comment:
You didn't fix the typo here that references React

Reply:
Fixed